types/dockerode/index.d.ts

```diff
@@ -1301,8 +1314,8 @@ declare class Dockerode {
     createService(options: {}, callback: Callback<any>): void;
     createService(options: {}): Promise<any>;
 
-    createNetwork(options: {}, callback: Callback<any>): void;
-    createNetwork(options: {}): Promise<any>;
+    createNetwork(options: {}, callback: Callback<Dockerode.Network>): void;
```

Comment:
Shouldn’t options be typed as NetworkCreateOptions here?

Reply:
Yes you're right, my bad. That's actually why I want this PR in the first place! The PR is closed so I'll have to create another unfortunately (I think).